test/katello_test_helper.rb

```diff
@@ -56,9 +56,13 @@ module FixtureTestCase
     self.pre_loaded_fixtures = true
 
     Katello::FixturesSupport.set_fixture_classes(self)
-    load_fixtures
-    self.fixture_path = "#{Katello::Engine.root}/test/fixtures/models"
+
+    fixture_path = Dir.mktmpdir("katello_fixtures")
+    FileUtils.cp(Dir.glob("#{Katello::Engine.root}/test/fixtures/models/*"), fixture_path)
+    FileUtils.cp(Dir.glob("#{Rails.root}/test/fixtures/*"), fixture_path)
+    self.fixture_path = fixture_path
```

Comment:
Does this need to be set after the copy versus as the first line in this set of statements?